src/http/states/readRequestLine/ParseMethod.cpp

```diff
@@ -0,0 +1,78 @@
+#include "ParseMethod.hpp"
+
+#include <client/Client.hpp>
+#include <http/Request.hpp>
+#include <http/StatusCode.hpp>
+#include <http/states/readRequestLine/ParseUri.hpp>
+#include <http/states/readRequestLine/ReadRequestLine.hpp>
+#include <libftpp/string.hpp>
+#include <utils/Buffer.hpp>
+#include <utils/BufferReader.hpp>
+#include <utils/abnfRules/LiteralRule.hpp>
+#include <utils/abnfRules/RangeRule.hpp>
+#include <utils/abnfRules/RepetitionRule.hpp>
+#include <utils/state/IState.hpp>
+#include <utils/state/StateHandler.hpp>
+
+#include <cctype>
```

Comment:
`::isdigit` is used, not `std::isdigit`.
```suggestion
#include <ctype.h>
```